datalad/interface/tests/test_run.py

```diff
@@ -766,6 +783,22 @@ def test_run_inputs_outputs(path):
     eq_(res["run_info"]['inputs'], ["a.dat"])
     eq_(res["run_info"]['outputs'], ["b.dat"])
 
+    # We install subdatasets to fully resolve globs.
+    ds.uninstall("s0")
+    assert_false(Dataset(op.join(path, "s0")).is_installed())
+    ds.run("echo {inputs} >globbed-subds", inputs=["s0/s1_*/s2/*.dat"])
+    ok_file_has_content(op.join(ds.path, "globbed-subds"),
+                        "s0/s1_0/s2/a.dat s0/s1_1/s2/c.dat",
```

Comment:
what will happen on Windows with all the paths here? just curious ;-)
I guess recorded `run` records would not be portable across platforms if we store them in "native" convention on Windows

Reply:
The tests have been incompatible with Windows from the beginning.  I plan to move away from the use of `echo`, etc, as well as hard-coded "/".  Then we can assess how much we need to change run/rerun to work on Windows, but I lean towards storing the paths in the commit message with "/" (like Git does) and then converting them when needed.